guide/getting-started/software.md

```diff
@@ -36,11 +36,13 @@ Illustration sources
 
 # Software overview
 
-Finance is complex, and there are many use cases, from giving weekly allowance to a child and selling goods in a store to managing multi-billion dollar retirement funds and hectic day trading. So we end up with many different application solutions for the various use cases. Bitcoin being a global currency adds another layer of complexity as the software may need to be adapted to different languages, cultures, and legal frameworks. When designing, it's a good idea to feel for this landscape and know where your product fits. Here is an overview of several common and specialized categories.
+There are countless use cases for financial tools, so there are almost as many application solutions per use case. Bitcoin’s global functionality adds complexity that may require adapting your application to foreign languages, cultures, and legal frameworks.
+Provided below is an overview of standard and specialized application categories.
+
 
 ## Command-line interfaces (CLI)
 
-Primarily used by developers, command-line interfaces typically offer the most feature-rich options to interact with blockchains. They can be thought of as general-purpose toolboxes, so they require certain technical knowledge and are not optimized for specific use cases. New features are often first included in a CLI and then later added to graphical user interfaces (GUI).
+Primarily used by developers, command-line interfaces offer feature-rich ways to interact with blockchains. They can be thought of as general-purpose toolboxes, so they require certain technical knowledge and are not optimized for specific use cases. New features are often first included in a CLI and then later added to graphical user interfaces (GUI).
```

Comment:
Adding comment by @sqcken for future reference:

>  They can be thought of as general-purpose toolboxes, so they require certain technical knowledge and are not optimized for specific use cases.

> Confusing. If they are general purpose, why do they require specific technical knowledge?

Reply:
Generic tools can be used in many ways. Doing specific things with them requires you to know exactly how to use the tools in the right way to get your desired results.

To give an example, I can use the general-purpose photo-editing tool Photoshop to apply cool filters to my photos, but then I really need to understand the complex UI, as well as color channel mixing, saturation, exposure, etc. Or I can use the specialized filters Instagram provides and be done in 5 seconds with 2 taps and not understanding what just happened.

Reply:
Ok, makes sense, got it 😄